serialization-avro/src/main/java/io/axoniq/dev/samples/serializationavro/api/GiftCardBusinessError.java

```diff
@@ -0,0 +1,11 @@
+package io.axoniq.dev.samples.serializationavro.api;
+
+import javax.annotation.Nonnull;
+
+public record GiftCardBusinessError(
```

Comment:
If we drop the `ExceptionWrappingHandlerInterceptor`, I think we can drop this as well.